frontend/src/metabase/palette/types.ts

```diff
@@ -5,19 +5,17 @@ import type { IconName } from "metabase/ui";
 
 interface PaletteActionExtras {
   extra?: {
-    /** parentCollection: Name of the collection to show in the palette item */
-    parentCollection?: string | null;
     /** isVerified: If true, will show a verified badge next to the item name */
     isVerified?: boolean;
-    /** database: Name of the database to show next the name in the palette item. */
-    database?: string | null;
     /**
      * href: If defined, the palette item will be wrapped in a link. This allows for
      * browser interactions to open items in new tabs/windows
      */
     href?: LocationDescriptor | null;
     /** iconColor: Color of the icon in the  */
```

Comment:
This isn't part of the diff but I noticed something was missing in this comment, worth throwing in a fix?